kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryImportServiceImpl.java

```diff
@@ -125,11 +126,11 @@ public int importMotherData(Reader reader) throws IOException {
                 importMotherRecord(record);
                 count++;
                 if (count % PROGRESS_INTERVAL == 0) {
-                    LOGGER.debug("Imported {}", timer.frequency(count));
+                    LOGGER.debug(IMPORTED, timer.frequency(count));
```

Comment:
Not sure if it was PMD or CPD. But the same constant was used 4 times and
it tripped an error, not just a warning.

On Mon, Sep 28, 2015 at 1:11 PM, Rob LaRubbio <notifications@github.com>
wrote:

> In
> kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryImportServiceImpl.java
> <https://github.com/motech-implementations/mim/pull/592#discussion_r40599161>
> :
>
> > @@ -125,11 +126,11 @@ public int importMotherData(Reader reader) throws IOException {
> >                  importMotherRecord(record);
> >                  count++;
> >                  if (count % PROGRESS_INTERVAL == 0) {
> > -                    LOGGER.debug("Imported {}", timer.frequency(count));
> > +                    LOGGER.debug(IMPORTED, timer.frequency(count));
>
> Did the CPD make you do this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/motech-implementations/mim/pull/592/files#r40599161>.
>



-- 
Frank Huster
Mobile Health Innovations
Grameen Foundation
+1.206.325.6690 x201